tests/boardfarm_plugins/boardfarm_prplmesh/tests/initial_ap_config.py

```diff
@@ -0,0 +1,65 @@
+
+import boardfarm
+
+from boardfarm.tests import bft_base_test
+from environment import sniffer, _get_bridge_interface
+
+
+class InitialApConfig(bft_base_test.BftBaseTest):
+    '''
+    Check initial configuration on device
+    '''
+
+    def runTest(self):
+        '''
+        Main test logic must be implemented here.
+        '''
+        regex_list = [
+            r"WSC Global authentication success",
+            r"KWA \(Key Wrap Auth\) success",
+            r".* Controller configuration \(WSC M2 Encrypted Settings\)"
+        ]
+        failed_regex = []
+
+        wired_sniffer = sniffer.Sniffer(_get_bridge_interface(self.dev.board.unique_id))
```

Comment:
Function `_get_gridge_interface` expected that docker network will have name `prplMesh-net-{unique_id}`, but in real test setup network may have another name template.
So i suggest 
1) set docker network name in test setup config
2) modify `_get_gridge_interface` to be able pass it docker network name instead of unique id